docker/vesting_contract/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM rust
+RUN rustup target add wasm32-unknown-unknown
+CMD cd nym/contracts/vesting && RUSTFLAGS='-C link-arg=-s' cargo build --release --target wasm32-unknown-unknown
```

Comment:
If you're already using docker, you might want to take a look at cosmwasm' image for producing smallest wasm blobs: https://github.com/cosmwasm/rust-optimizer
Unless it's a quick change, I don't think you really need to do it now

Reply:
I'll do this in a separate change for both vesting and mixnet :+1:

Reply:
I'll need to have a quick glance over it.